R/derive_vars_transposed.R

```diff
@@ -7,8 +7,9 @@
 #'
 #' @param dataset_merge Dataset to transpose and merge
 #'
-#'   The variables specified by the `by_vars`, `key_var` and `value_var` parameters
-#'   are expected
+#'   The variables specified by the `by_vars`, `id_vars`, `key_var` and
+#'   `value_var` arguments are expected. The variables `by_vars`, `id_vars`,
+#'   `key_var` have to be a unique key.
```

Comment:
Should we say anything about an Error will be produced from signal_duplicates if the variables in those arguments are not unqiue?

Reply:
I think usually we don't state explicitly that an error is issued if the input is not as expected. Thus I wouldn't add it here.

Reply:
I love conventions!